app/src/main/jni/cmake/FindBoost.cmake

```diff
@@ -1,7 +1,7 @@
 set(Boot_FOUND TRUE)
 
 # libraries that can't compile
-set(BOOST_EXCLUDE_LIBRARIES "context;coroutine;fiber" CACHE STRING "" FORCE)
+set(BOOST_EXCLUDE_LIBRARIES "fiber" CACHE STRING "" FORCE)
```

Comment:
How about also including `fiber` in `BOOST_INSTALLED_LIBRARIES`?

Reply:
I can build with fiber included, and I just changed the RimePlugins.cmake as you suggested.